index.js

```diff
@@ -8,38 +8,32 @@ const gunzip = util.promisify(zlib.gunzip);
 const { getChangesets } = require('./lib/get-changesets');
 const { uploadToS3 } = require('./lib/s3');
 const { postTagChanges } = require('./lib/tagChanges');
+const { formatReplicationKey } = require('./util/format-replication-key');
+const { request } = require('./util/request');
 const { s3 } = require('./lib/s3-client');
 
 process.on('unhandledRejection', (up) => { throw up; });
 process.on('exit', (code) => {
     console.log(`Exit with code: ${code}`);
 });
 
-const snsSubject = process.env.Subject;
-const snsMessage = JSON.parse(process.env.Message);
-const AMAZON_S3_NOTIFICATION = 'Amazon S3 Notification';
+const run = async (replicationFileId) => {
+  console.time('TIME');
+  const { Body } = await s3.getObject({
+    Bucket: 'osm-planet-us-west-2',
```

Comment:
@willemarcel not so important to fix right now, but can we make this configurable as well? This would allow us to use the same code for projects like OHM.

Reply:
Done!